packages/analytics-js-integrations/__tests__/integrations/VWO/browser.test.js

```
@@ -141,3 +142,49 @@ describe('VWO Identify Event', () => {
     );
   });
 });
+
+describe('VWO SmartCode v3 loader sanity', () => {
+  const ACCOUNT_ID_THRESHOLD = 1200000;
+
+  beforeEach(() => {
+    jest.useFakeTimers();
+    delete window._vwo_code;
+    // SmartCode v3 assigns `code` without declaration; declare it for strict-mode Jest runtime
+    globalThis.eval('var code;');
```

Biome (2.4.9)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)